### DISCUSSION TITLE
popout a statblock in a new window

### DISCUSSION THREAD

**Discussion by Lombax99** [Obsidian-TTRPG-Community/fantasy-statblocks#273] [Feature Requests]
I'd like to see multiple statsblock at the same time from the creature pane tab. Not necessarily in the tab itself, I was thinking of some way of searching a stat block, opening it in a window or tab, and leaving you able to search for a second stat block. 
This way one could keep any number of monsters open at the same time.

**Comment by valentine195** [comment]
Added in #275

**Reply by valentine195** [reply]
Specifically, you can just drag creature panes out of Obsidian to open them in new windows. #275 allows you to open multiple panes.
